test/functional/feature_assumeutxo.py

```diff
@@ -156,19 +156,19 @@ def no_sync():
         self.connect_nodes(0, 1)
 
         self.log.info(f"Ensuring snapshot chain syncs to tip. ({FINAL_HEIGHT})")
-        wait_until_helper(lambda: n1.getchainstates()['snapshot']['blocks'] == FINAL_HEIGHT)
+        self.wait_until(lambda: n1.getchainstates()['snapshot']['blocks'] == FINAL_HEIGHT, timeout=120)
```

Comment:
@MarcoFalke Did you see @achow101 's comment: https://github.com/bitcoin/bitcoin/pull/27596#discussion_r1343167013 ?

Reply:
@ryanofsky I think that's an interesting idea to set a test-wide default for the `wait_until` helper, I implemented this [here](https://github.com/bitcoin/bitcoin/pull/28562/commits/bd5e9c5cd84ffd0803bf9b8b35ec15c7afe6554f) for now but it may be better to pull it out into its own PR... What do you think about this idea @MarcoFalke ?